hal/src/stm32f2xx/core_hal_stm32f2xx.c

```diff
@@ -426,6 +427,43 @@ bool HAL_Core_Validate_User_Module(void)
     }
     return valid;
 }
+
+bool HAL_Core_Validate_Modules(uint32_t flags, void* reserved)
+{
+    const module_bounds_t* bounds = NULL;
+    hal_module_t mod;
+    bool module_fetched = false;
+    bool valid = true;
```

Comment:
Fixed in eedde55c7beddd134864a3974703ff55e5b2e9f6, however the default in `validate_module_dependencies_full` has to be left `true`.